src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -224,4 +227,20 @@ describe('<CourseEnrollments />', () => {
     wrapper.find('.alert .btn.close').simulate('click');
     expect(mockModifyIsMarkCourseCompleteSuccess).toBeCalledTimes(1);
   });
+
+  it('properly closes mark course as archived success status alert', () => {
+    const enterpriseConfig = {
+      uuid: 'test-enterprise-uuid',
+    };
+    const wrapper = mount((
```

Comment:
oh wow a proper mount. I've only ever used `shallow` 😂

Reply:
yup I think with enzyme shallow was used a lot. I think RTL encourages (or even just defaults to full mounting and rendering... I have to dig deeper into render but I am pretty sure it does that). this FAQ answers some of what the RTL render() does and how to achieve shallow mount behavior if desired ...  https://testing-library.com/docs/react-testing-library/faq. doing full does require more work in mocking and handling side effects .. but it also give more realistic testing .. we will have to see case by case I guess ... :)